aten/src/ATen/native/quantized/cpu/qnnpack/src/qnnpack/requantization.h

```diff
@@ -55,6 +55,31 @@ pytorch_qnnp_compute_scalar_requantization_params(
   return params;
 }
 
+static inline union pytorch_qnnp_fp32_requantization_params
+pytorch_qnnp_compute_scalar_fp32_requantization_params(
+    float scale,
+    uint8_t zero_point,
+    uint8_t min,
+    uint8_t max) {
+  /* Compute requantization parameters */
+  assert(scale < 1.0f);
```

Comment:
is this the requantization scale? Why do we have this check here?

Reply:
So my plan was to first land changes for doing requantization via FP first. So that is one PR. And then we switch off the scale constraint in a separate PR, so it is easy to revert if necessary.